src/overtone/sc/sclang.clj

```diff
@@ -0,0 +1,390 @@
+(ns overtone.sc.sclang
+  (:require
+   [babashka.process :as proc]
+   [clojure.edn :as edn]
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [overtone.helpers.system :refer [get-os linux-os? mac-os? windows-os?]]
+   [overtone.sc.machinery.server.connection :as ov.conn]
+   [overtone.sc.synth :as ov.synth]))
+
+(defn sclang-path
+  "The user must have SuperCollider installed.
+
+  For sclang location in each OS, see
+  https://github.com/supercollider/supercollider/wiki/Path-searching#scide-finds-sclang"
+  []
+  (if (mac-os?)
+    "/Applications/SuperCollider.app/Contents/MacOS/sclang"
+    ;; It's simply `sclang` in non OSX environments,
+    ;; https://github.com/supercollider/supercollider/wiki/Path-searching#scide-finds-sclang
+    ;; TODO Someone should test on Linux and Windows to confirm that this works.
+    "sclang"))
+
+(defn transpile
```

Comment:
We can add more documentation here in the future, but for now, the users that will use this should have a knowledge of SC's lang anyway and I think this should be the internal representation anyway. The plan includes making it more clojure-like in the future.